.appveyor.yml

```diff
@@ -11,7 +11,9 @@ branches:
 environment:
   GOPATH: C:\gopath
   CGO_ENABLED: 1
-  GO_VERSION: 1.9
+  matrix:
+    - GO_VERSION: 1.9
+    - GO_VERSION: 1.10rc1
```

Comment:
Looks like this is the way to do it in AppVeyor; https://www.appveyor.com/docs/build-configuration/#build-matrix